tiflash/troubleshoot-tiflash.md

```diff
@@ -127,12 +127,12 @@ After deploying a TiFlash node and starting replication (by performing the ALTER
     - If there is output, go to the next step.
     - If there is no output, run the `SELECT * FROM information_schema.tiflash_replica` command to check whether TiFlash replicas have been created. If not, run the `ALTER table ${tbl_name} set tiflash replica ${num}` command again, check whether other statements (for example, `add index`) have been executed, or check whether DDL executions are successful.
 
-2. Check whether the TiFlash process runs correctly.
+2. Check whether TiFlash Region replication runs correctly.
 
-   Check whether there is any change in `progress`, the `flash_region_count` parameter in the `tiflash_cluster_manager.log` file, and the Grafana monitoring item `Uptime`:
+   Check whether there is any change in `progress`:
 
-   - If yes, the TiFlash process runs correctly.
-   - If no, the TiFlash process is abnormal. Check the `tiflash` log for further information.
+   - If yes, TiFlash replication runs correctly.
+   - If no, TiFlash replication is abnormal. In `tidb.log`, search the log saying `Tiflash replica is not available`. Check whether `region have` of the corresponding table is updated. If not, check the `tiflash` log for further information.
```

Comment:
The `region have` has been changed into `progress` in master.

Reply:
I will create a PR to update the Chinese.

Reply:
https://github.com/pingcap/docs-cn/pull/13101